pandas/core/reshape/concat.py

```diff
@@ -344,6 +344,22 @@ def concat(
     Traceback (most recent call last):
         ...
     ValueError: Indexes have overlapping values: ['a']
+
+    Append a single row to the end of a ``DataFrame`` object.
+
+    >>> a = pd.DataFrame({"A": 1, "B": 2}, index=[0])
+    >>> a
+        A   B
+    0   1   2
+    >>> b = pd.DataFrame({"A": 3}, index=[0])
+    >>> b
+        A
+    0   3
+    >>> for rowIndex, row in b.iterrows():
+    >>>     print(pd.concat([a, row.to_frame().T], ignore_index=True))
```

Comment:
does it work to make this
```
new_row = pd.Series([3])
```
and then just do
```
pd.concat([a, new_row.to_frame().T], ignore_index=True)
```
?
Iterating over rows is what we want to avoid

Also, let's rename `a` to `df7`

Reply:
Changed as requested in 90bafc6d3b223ae8791e631dac582d206b922842

Is it ok that the data from the new_row have appeared in a new row and not in the first one as in the previous solution?

I've also read the discussion about the deprecated append and maybe we should add some note into the documentation as well, something like 

> It is not recomended to build DataFrames by adding single rows in a
not loop. Build a list of rows and make a DataFrame in a single concat.